model/location.rb

```diff
@@ -52,6 +52,26 @@ def aws?
   def pg_ami(pg_version, arch)
     PgAwsAmi.find(aws_location_name: name, pg_version:, arch:).aws_ami_id
   end
+
+  # only valid for aws locations
+  def azs
+    unless aws?
+      Clog.emit("azs is only valid for aws locations") { {location_id: id} }
```

Comment:
Should this method ever be called for non-AWS?  If not, I think it's better to raise instead of using Clog.emit.